plugin/src/main/groovy/com/novoda/buildproperties/BuildPropertiesPlugin.groovy

```diff
@@ -99,14 +95,4 @@ class BuildPropertiesPlugin implements Plugin<Project> {
             })
         }
     }
-
-    private static void addSigningConfigSupportTo(target) {
```

Comment:
didn't know it was a thing :D I'm pretty sure I used the more basic features of BPP to store this stuff